integration/integrate-with-image-vulnerability-scanners.adoc

```diff
@@ -30,11 +30,20 @@ This enhanced support gives you greater flexibility and choice in managing your
 [discrete]
 == Supported Scanners
 
-You can set up {product-title-short} to obtain image vulnerability data from the following commercial container image vulnerability scanners:
+You can set up {product-title-short} to obtain image vulnerability data from the following commercial container image vulnerability scanners.
```

Comment:
I was coming here to say that it seemed like this should be a colon here, and to get rid of the following line. But I can see now why you added it.

Maybe you could keep the colon here, and then add those two lines "Scanners included in {product-title-short}:" and "Alternative scanners:" as headings?